### PR TITLE
Use new SDK 3.0.0 in demo app。

### PR DESCRIPTION
**Pull request by zhangcc01**
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [x] Did you check UI tests on the sample app? They are not executed on CI.

## Description

Use new SDK 3.0.0 in demo app。

### CODE REVIEW

File: apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java [hunk 1]
```diff
@@ -113,13 +113,7 @@ static void startAppCenter(Application application, String startTypeString) {
         int savedTrack = sSharedPreferences.getInt(application.getString(R.string.appcenter_distribute_track_state_key), 0);
         if (savedTrack != 0) {
             try {
```
Comment (annakocheshkova) [inline apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java]: Please remove try {... } catch(..) around the call

Comment (zhangcc01) [reply apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java]: @annakocheshkova Yes, I have removed in new commit.

File: apps/sasquatch/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -65,9 +65,6 @@
         <activity
             android:name=".activities.AuthenticationProviderActivity"
             android:label="@string/title_auth" />
-        <activity
```
Comment (guperrot) [inline apps/sasquatch/src/main/AndroidManifest.xml]: That's a merge of the 2 activity Java files between build variants but the file still exists. You cannot remove the one reference to the manifest.

Comment (zhangcc01) [reply apps/sasquatch/src/main/AndroidManifest.xml]: @guperrot Updated in new commit.

**Pull request merged**